src/scene/mesh.js

```diff
@@ -497,6 +497,48 @@ Object.assign(pc, function () {
             // Deprecated
         },
 
+        destroy: function() {
+            
+            var mesh,  ib,  j;
+            var device;
+            
+            mesh = this.mesh;
+            if (mesh) {
+                mesh._refCount--;
+                if (mesh._refCount < 1) {
+                    if (mesh.vertexBuffer) {
+                        device = device || mesh.vertexBuffer.device;
+                        mesh.vertexBuffer.destroy();
+                        mesh.vertexBuffer = null;
+                    }
+                    for (j = 0; j < mesh.indexBuffer.length; j++) {
+                        device = device || mesh.indexBuffer.device;

```

Comment:
```device``` is not needed. Can be deleted.

Reply:
ok,I repaired it.I found that the set function of mesh may not be released when the mesh reference count is reduced to zero.